tests/testthat/test-process_content.R

```diff
@@ -1,48 +1,43 @@
-test_that("processing content have proper output structure", {
-
-  my_project <-
-    initialize_project("gitai_test_project") |>
+test_that("[integration test] processing content have proper output structure", {
+  my_project <- initialize_project("gitai_test_project") |>
     set_llm() |>
     set_prompt(system_prompt = "Say 'Hi there!' only and nothing else.")
 
   result <- process_content(gitai = my_project, content = "")
```

Comment:
Tests locally fail here, as `turn` object is `NULL`.  It is created with `last_turn()` method, not sure if it should be somehow mocked?

Reply:
It happened due to overwriting elmer:::Chat (indirectly by modification of ChatMocked).
Fixed it by using inherited R6 Class.